src/pages/rank/TopCard.tsx

```diff
@@ -0,0 +1,37 @@
+import { Avatar, Typography } from '@mui/material';
+import React from 'react';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+interface TopCardProps<T extends Record<string, any>> {
+  item: T;
+  message: string;
+  index: number;
+}
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const TopCard = <T extends Record<string, any>>({ item, message, index }: TopCardProps<T>) => {
+  return (
+    <div key={item.id} className="relative h-40 w-full">
+      <div className="absolute h-full w-full pb-4 pr-4">
+        <div className="flex h-full w-full bg-mainBlack p-4">
+          <div className="flex h-full w-full flex-col justify-between">
+            <div className="flex">
+              <Typography className="border-[1px] border-pointBlue px-3 py-1 text-xl font-semibold text-pointBlue">
+                {index + 1}
+              </Typography>
+              <Typography className="my-auto ml-2 text-2xl font-bold">{item.name}</Typography>
+              <Typography className="my-auto ml-2">{item.no}기</Typography>
+            </div>
+            <Typography className="font-semibold text-pointBlue">{message}</Typography>
+          </div>
+          <Avatar alt="profile" className="mr-2 !h-20 !w-20 object-contain" />
+        </div>
+      </div>
+      <div className="h-full w-full pl-4 pt-4">
+        <div className="h-full w-full border-2 border-pointBlue" />
+      </div>
+    </div>
```

Comment:
```suggestion
    <div className="relative h-full">
      <div className="absolute z-10 h-28 w-80">
        <div className="flex h-full justify-between bg-mainBlack p-4">
          <div className="flex h-full flex-col justify-between">
            <div className="flex">
              <Typography className="border-[1px] border-pointBlue px-3 py-1 text-pointBlue">{index + 1}</Typography>
              <Typography variant="h3" fontWeight="semibold" marginY="auto" marginLeft={1}>
                {item.name}
              </Typography>
              <Typography variant="small" marginY="auto" marginLeft={1}>
                {item.no}기
              </Typography>
            </div>
            <Typography color="primary" fontWeight="semibold">
              {message}
            </Typography>
          </div>
          <Avatar alt="profile" className="my-auto !h-16 !w-16" />
        </div>
      </div>
      <div className="absolute left-2 top-2 h-28 w-80 border-2 border-pointBlue" />
    </div>
```

이거 제가 그냥 디자인 맞춰본다고 해본 건데 해본 김에 공유드립니다! 아바타 크기와 위에 코멘트 남겼던 폰트 적용한 것과 카드 크기 정도 디자인 맞춰본 것 같아요! (이거 그냥 가져다 사용하셔도 되고 도균님 스타일에 맞게 변형해서 사용해도 됩니당!)